README.adoc

```diff
@@ -174,24 +228,101 @@ Same as in the `ne4j` repo.
 Suggestions for improvement::
 
 Give a possible solution for the provided example query.
+====
 
-⛔Example:
+For example:
+
+[options="header",cols="h,2,2",]
+|===
+| Example
+| ⛔ Old example
+| ✅ New example
```

Comment:
Same as above, I think 'Old' and 'New' example does not make much sense here, as we only have new examples.